docs/fundamentals/write-operations.txt

```diff
@@ -0,0 +1,608 @@
+
+   DB::connection('mongodb')
+      ->getCollection(<collection name>)
+      ->updateOne(
+          <match criteria>,
+          ['$set' => ['<array field>.$' => <replacement value>]]);
+
+
+The following example shows how to replace the array value ``"dance-pop"``
+with ``"contemporary"`` in the ``genres`` array field. Click the
+:guilabel:`VIEW OUTPUT` button to see the updated document:
+
+.. io-code-block::
+
+   .. input:: /includes/fundamentals/write-operations/WriteOperationsTest.php
+      :language: php
+      :dedent:
+      :start-after: begin model array positional
+      :end-before: end model array positional
+
+   .. output::
+      :language: json
+      :visible: false
+
+      {
+        "_id": "660e...",
+        "performer": "Mitsuko Uchida",
+        "genres": [
+          "classical",
+          "contemporary"
+        ],
+        "updated_at": ...,
+        "created_at": ...
+      }
+
+To learn more about array update operators, see :manual:`Array Update Operators </reference/operator/update-array/>`
+in the {+server-docs-name+}.
+
+.. _laravel-fundamentals-delete-documents:
+
+Delete Documents
+----------------
+
+In this section, you can learn how to delete documents from a MongoDB collection
+by using {+odm-short+}. Use delete operations to remove data from your MongoDB
+database permanently.
+
+This section provides examples of the following delete operations:
+
+- :ref:`Delete one document <laravel-fundamentals-delete-one>`
+- :ref:`Delete multiple documents <laravel-fundamentals-delete-many>`
+
+To learn about the Laravel features available in {+odm-short+} that modify
+delete behavior, see the following sections:
+
+- :ref:`Soft delete <laravel-model-soft-delete>`, which lets you mark
+  documents as deleted instead of removing them from the database
+- :ref:`Pruning <laravel-model-pruning>`, which lets you define conditions
+  in which a document should be deleted automatically
+
+.. _laravel-fundamentals-delete-one:
+
+Delete a Document Examples
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can delete one document in the following ways:
+
+- Call the  ``destroy()`` method on the model to delete a document by the
+  primary key.
+- Call the ``delete()`` method on an instance of the model
+- Chain methods to retrieve and delete an instance of a model by calling the
+  ``delete()`` method.
+
+The following example shows how to delete a document by passing the value of
+its primary key to the ``destroy()`` method:
+
+.. literalinclude:: /includes/fundamentals/write-operations/WriteOperationsTest.php
+   :language: php
+   :dedent:
+   :start-after: begin model delete by id
+   :end-before: end model delete by id
+
+When the ``destroy()`` method succeeds, it returns the number of documents
+deleted.
+
+If the primary key value does not match any documents, the ``destroy()`` method
+returns returns  ``0``.
+
+The following example shows how to delete a document by calling ``delete()``
+on an instance of the model:
```

Comment:
S: separate the three examples into subsections or move into bullet points, but as of now stacking them is a bit confusing

Reply:
I agree with this and had considered earlier on, but wanted to keep all examples at the same heading level. Both Insert and Update use the same formatting with the same intention.

I think once these sections (Insert, Modify, Delete) are split into individual pages similar to drivers docs are separate pages,  these can be their own sections.

In the meantime, I'll add literalinclude captions (including examples in the prior sections) to identify which method is being demonstrated.